plugin/proto.go

```diff
@@ -26,6 +32,11 @@ const (
 // Command is a CLI command available in the plugin contract.
 type Command string
 
+// Request defines a plugin request, which is always associated to a command.
+type Request interface {
+	Command() Command
+}
```

Comment:
What's the purpose of this interface?

Reply:
Well, technically we shouldn't need this but the way its currently implemented in [notation-go](https://github.com/search?q=repo%3Anotaryproject%2Fnotation-go%20Command&type=code) we need this interface.